R/stat_likelihoods.R

```diff
@@ -86,43 +86,61 @@ geom_length_ll <- function(x, prob) {
   log(GkmGkm1)
 }
 
-#' Likelihood of the length of chains with generic offspring distribution
+#' Log-likelihood of the summary (size/length) of chains with generic offspring
+#' distribution
 #'
-#' The likelihoods are calculated with a crude approximation using simulated
-#' chains by linearly approximating any missing values in the empirical
+#' The log-likelihoods are calculated with a crude approximation using simulated
+#' chain summaries by linearly approximating any missing values in the empirical
 #' cumulative distribution function (ecdf).
 #' @inheritParams likelihood
-#' @inheritParams simulate_vec
-#' @param chains Vector of sizes/lengths
+#' @inheritParams simulate_summary
+#' @param chains Vector of chain summaries (sizes/lengths)
 #' @param nsim_offspring Number of simulations of the offspring distribution
-#' for approximating the statistic (size/length) distribution
-#' @param log Logical; Should the results be log-transformed? (Defaults
-#' to TRUE).
-#' @param ... any parameters to pass to \code{\link{simulate_tree}}
-#' @return If \code{log = TRUE} (the default), log-likelihood values,
-#' else raw likelihoods
+#' for approximating the distribution of the chain statistic summary
+#' (size/length)
+#' @param ... any parameters to pass to \code{\link{simulate_summary}}
+#' @return log-likelihood values
 #' @author Sebastian Funk
 #' @export
+#' @seealso [simulate_summary()] for simulating a summary of the transmission
+#' chains statistic (without the tree of infections)
+#' @examples
+#' set.seed(123)
+#' chain_size_ll <- offspring_ll(
+#'   chains = c(1, 5, 6, 8, 7, 8, 10),
+#'   offspring_dist = "pois",
+#'   statistic = "size",
+#'   lambda = 0.82
+#' )
 offspring_ll <- function(chains, offspring_dist, statistic,
```

Comment:
would it make sense for `chains` to be called `x` as above? Would have to re-document `chains` in the likelihood function then as it couldn't be inherited.

Reply:
Fixed in a9db3fcf46e8c255953b90ca357901a161231211.